subworkflows/local/clonal_analysis.nf

```diff
@@ -30,15 +30,36 @@ workflow CLONAL_ANALYSIS {
             ch_logo,
             ch_find_threshold_samplesheet
         )
-        ch_threshold = FIND_CLONAL_THRESHOLD.out.mean_threshold
+        def ch_threshold_channel = FIND_CLONAL_THRESHOLD.out.mean_threshold
```

Comment:
Nitpicking comment. If "ch_" stands for channel, then maybe "_channel" can be removed.